test/parallel/test-http-unix-socket-keep-alive.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const http = require('http');
+
+const server = http.createServer((req, res) => res.end());
+
+common.refreshTmpDir();
+
+server.listen(common.PIPE, () =>
```

Comment:
I guess I'll put them in anyway to be super clear that these are all expected to run.